openwisp_notifications/tests/test_notifications.py

```diff
@@ -822,3 +829,17 @@ def test_post_migrate_handler_celery_broker_unreachable(self, mocked_logger, *ar
             sender=NotificationsAppConfig, app_config=NotificationsAppConfig
         )
         mocked_logger.assert_called_once()
+
+    def test_notification_cache_update(self):
+        operator = self._get_operator()
+        self._register_notification_cache_update(User, post_save, 'post_save')
+        self.notification_options.update(
+            {'action_object': operator, 'target': operator, 'type': 'default'}
+        )
+        self._create_notification()
+        content_type = ContentType.objects.get_for_model(operator._meta.model)
+        operator.username = "new operation name"
+        operator.save()
+        cache_key = Notification._cache_key(content_type.id, operator.id)
+        _cache = cache.get(cache_key, None)
```

Comment:
there is this assertion which checks that the cache has been emptied but there's no assertion which checks the cache contains anything, or am I wrong? We should have an assertion that ensures the cache contains the right data

Reply:
Yes, you are right. Actually I was thinking deletion of cache is important in this test. 
I will add test to ensures that the cache contains the right data